test/Prompt/DontWrap.C

```diff
@@ -172,7 +172,7 @@ const Test2::Nested::Value::type & Test2::Nested::reference() const {
 Test2::Nested Nest2(80);
 // CHECK: Nested::Nested(160)
 Nest2.simpleAdd(3)
-// CHECK: (Test2::Nested::Value::type) 83
+// CHECK: (int) 83
```

Comment:
> Eg., if we have:
> [cling]$ tb2.begin()
> ((std::__1::vector<Test, std::__1::allocator<Test> >::iterator)) aka [std::vector<Test, . std::allocator<Test> >::iterator] @0x7fad5aa0ce80
>
> would be fine to write the tests on Windows because FileCheck is smart and it will scan > only for the aka part.

I strongly disagree with @vgvassilev.  Having nice name whenever possible on a prompt for human consumption is *much* better.  I think the behavior from @marsupial displayed above following "From this thread I have another commit that does this:" is good/desirable.

*If* the user need to know the gorry detail, let's just have a '.desugar typedefname' command.  thanks.